content/applications/hr/payroll.rst

```diff
@@ -526,34 +696,35 @@ Resume
 
 .. note::
```

Comment:
Not sure if this is just because I was using Runbot, but I was able to access the Resume page fairly easily. Just thought it was worth noting, just in case it actually is now available in non-Belgian markets

Reply:
OMG they added this in the recent release, but it wasn't listed in the release notes =D.  That's because this is tied into the US localization! I added the whole new section here =).